src/hooks/useWearables.ts

```diff
@@ -110,5 +72,54 @@ export const useSetWearableState = () => {
       .then((res) => res.data);
   };
 
-  return setWearableState;
+  const setLastSync = async ({ ehrId, lastSync, failureCode }: SetLastSync) =>
+    httpClient.patch(
+      `/v1/wearables/${ehrId}`,
+      {
+        lastSync,
+        status: failureCode
+          ? WearableIntegrationStatus.Failure
+          : WearableIntegrationStatus.Syncing,
+        failureCode,
+      },
+      { headers: accountHeaders },
+    );
+
+  const setSyncTypes = async (settings: SyncTypeSettings) =>
+    httpClient.post('/v1/wearables/sync-types', settings, {
+      headers: accountHeaders,
+    });
+
+  const useWearableIntegrationQuery = (ehrId: string) =>
+    useQuery(
+      'get-wearable',
+      () =>
+        httpClient.get(`/v1/wearables/${ehrId}`, { headers: accountHeaders }),
+      {
+        enabled: !!accountHeaders,
+      },
+    );
+
+  const useWearableIntegrationsQuery = (appId: string) =>
+    useQuery(
+      'get-wearables',
+      () =>
+        httpClient
+          .get<WearablesSyncState>('/v1/wearables', {
+            params: { appId },
+            headers: accountHeaders,
+          })
+          .then((res) => res.data),
+      {
+        enabled: !!accountHeaders,
+      },
+    );
+
+  return {
+    setWearableState,
```

Comment:
I attempted to utilize useMutation here but it would lead to quite a bit of refactor in the Wearables components so I decided to punt on that for now.

Reply:
👍 In the fhir client where I'm utilizing mutation, I'm still figuring out how to make error handling actually work.  I also find myself consistently utilizing `mutateAsync`.  So the async func approach you have in place might be altogether better for now.